SmartPlaces.Facilities/products/Topology/src/ProcessTopology.cs

```diff
@@ -0,0 +1,53 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="ProcessTopology.cs" company="Microsoft">
+//   Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace Topology
+{
+    using System.Threading.Tasks;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.Hosting;
+    using Microsoft.Extensions.Logging;
+    using IngestionManager.Interfaces;
+
+    public class ProcessTopology : BackgroundService
+    {
+        private readonly ILogger<ProcessTopology> logger;
+        private readonly IConfiguration configuration;
+        private readonly IGraphIngestionProcessor graphIngestionProcessor;
+
+        public ProcessTopology(ILogger<ProcessTopology> logger,
+                               IConfiguration configuration,
+                               IGraphIngestionProcessor graphIngestionProcessor)
+        {
+            this.logger = logger;
+            this.graphIngestionProcessor = graphIngestionProcessor;
+            this.configuration = configuration;
+
+            logger.LogInformation("AzureDigitalTwinsEndpoint: {AzureDigitalTwinsEndpoint}", configuration["AzureDigitalTwinsEndpoint"]);
+            logger.LogInformation("StorageAccountEndpoint: {StorageAccountEndpoint}", configuration["StorageAccountEndpoint"]);
```

Comment:
Didn't this get removed with the bulk PR?

Reply:
It's everywhere!